flowsa/data_source_scripts/BTS_TSA.py

```diff
@@ -0,0 +1,71 @@
+# BEA.py (flowsa)
+# !/usr/bin/env python3
+# coding=utf-8
+
+'''
+Pulls BTS Transportation Satellite Account (TSA) data
+'''
+from io import BytesIO
+import pandas as pd
+from flowsa.common import US_FIPS, fbs_activity_fields
+from flowsa.schema import activity_fields
+from flowsa.settings import externaldatapath
+from flowsa.flowbyfunctions import assign_fips_location_system
+from flowsa.fbs_allocation import allocation_helper
+
+
+def tsa_call(*, resp, year, **_):
+    '''
+    Convert response to pandas dataframe
+    :param resp: response from url call
+    :return: pandas dataframe of original source data
+    '''
+    df = pd.read_csv(BytesIO(resp.content))
+    df = df[df.year == year]
```

Comment:
@matthewlchambers  I think year is a string in the function, but in your df is an int

Reply:
Dang. Good eye! That was indeed the problem. In testing I was calling the `tsa_call()` function separately, with `year` as an int. Hence why it appeared that the problem was downstream from here.